docs/AccordionForm.md

```diff
@@ -351,7 +351,7 @@ export const TagEdit = () => (
 );
 ```
 
-**Warning**: This feature only works if you have a dependency on react-router 6.3.0 **at most**. The react-router team disabled this possibility in react-router 6.4, so `warnWhenUnsavedChanges` will silently fail with react-router 6.4 or later.
+**Note**: Due to limitations in react-router, this feature only works if you use the default router provided by react-admin, or if you use a Data Router.
```

Comment:
nitpick: I'd add a link to react-router's documentation for those not knowing what a Data Router is

Reply:
you're right, added.